docs/moments.rst

```diff
@@ -62,6 +62,28 @@ plotting packages such as APLpy::
     >>> f.show_colorscale()  # doctest: +SKIP
     >>> f.save('moment_0.png')  # doctest: +SKIP
 
+The equation for the N-th moment, where N is usually an integer between -1 and some arbitrary
+positive integer say 11, is as follows:
+
+.. math:: M_N = \frac{\int I (l - M_l)^N dl}{M_0}
+
+Descriptions for the twelve most common moments used are as follows:
+
+moments=-1 - the mean value of the spectrum  
+moments=0  - the integrated value of the spectrum, can be used for computing line ratios
+moments=1  - the intensity weighted coordinate; by convention used to get the ’velocity fields’
+moments=2  - the intensity weighted dispersion of the coordinate; by convention used to get the ’velocity dispersion’  
+moments=3  - the median of the cube I
```

Comment:
This and all subsequent moments are incorrect.  These are the convention of a specific version of another piece of software and they do not apply to spectral-cube.  I very strongly discourage the use of these 'moment' values, since they are an incorrect usage of the term.